holoviews/plotting/mpl/renderer.py

```diff
@@ -318,3 +321,4 @@ def load_nb(cls, inline=True):
         """
```

Comment:
A parameter doesn't make much sense to me because it means it can be changed at the instance level which doesn't seem quite correct. I think it's fine as is tbh because I'd still have to make the property settable somehow.